addons/html_editor/static/src/main/table/table_plugin.js

```diff
@@ -370,6 +374,128 @@ export class TablePlugin extends Plugin {
         }
         this.dependencies.selection.setSelection(selectionToRestore);
     }
+
+    /**
+     * @param {HTMLTableRowElement} row
+     */
+    normalizeRowDimension(row) {
+        const table = closestElement(row, "table");
+        const rows = [...table.rows];
+        const referenceRowHeight = parseFloat(getComputedStyle(row).height);
+        rows.forEach((row) => {
+            if (
+                row.style.height &&
+                Math.abs(parseFloat(row.style.height) - referenceRowHeight) <= 1
+            ) {
+                row.style.height = "";
+            }
+        });
+    }
```

Comment:
This function is weird. From its name and parameter you would think it only affect the "dimension" (which dimension?) of the provided row, but in fact it affect all the rows of the table and affect their _height_. Wouldn't `normalizeRowHeight(table)` make more sense?